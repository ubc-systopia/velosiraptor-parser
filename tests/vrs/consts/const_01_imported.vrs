import const_00_defs;

const FOOBAR: int = 1234;
const FOOBAR: int = 0xabcd;