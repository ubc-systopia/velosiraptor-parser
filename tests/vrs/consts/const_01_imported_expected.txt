VelosiParseTree(tests/vrs/consts/const_01_imported.vrs)
---------------------------------------------
const FOO : int = 123;

const FOO : int = 4660;

const FOO : int = a;

const FOO : int = BAR;

const FOO : size = FOO + FOOBAR;

const FOO : MyType = 1234;

const FOO : bool = 1234;

const FOO : int = true;

const FOOBAR : int = 1234;

const FOOBAR : int = 43981;

---------------------------------------------
